.changeset/loud-swans-repeat.md

```diff
@@ -0,0 +1,8 @@
+---
+'@primer/react-brand': patch
+---
+
+Style improvements to the `SubNav` component
+
+- A `SubNav.Link` with `aria-current` will now be underlined to indicate the current page
```

Comment:
Just to check, the a11y indicators are already there but you've made them wider now? Is that correct?

Reply:
Yes that's true, although there were a gaps in the implementation where they wouldn't appear, or would disappear on hover. Specifically when a page inside a SubMenu was the current page. On mobile, the underline would disappear when anything in the SubMenu (or the parent) was hovered, and on desktop it wouldn't show at all.

Appreciate this changeset could be a bit clearer in that regard, happy to update it if you have a suggestion of how to clarify?

Reply:
I've tweaked the changeset, lmk if you have any suggestions